tests/test_file_svg.py

```diff
@@ -38,3 +38,20 @@ def test_quick():
         assert im.size == (24, 24)
         assert_image_similar(im, compare, 1)
```

Comment:
BTW I had to change this to `(im, compare, 27)` to pass the unit-test on clean `master` branch.